src/setup-ros-linux.ts

```diff
@@ -101,11 +105,19 @@ export async function runLinux() {
 		"-c",
 		`echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros-latest.list`,
 	]);
-	await utils.exec("sudo", [
-		"bash",
-		"-c",
-		`echo "deb http://packages.ros.org/ros2/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros2-latest.list`,
-	]);
+	if(use_ros2_testing) {
+          await utils.exec("sudo", [
+  		"bash",
+  		"-c",
+  		`echo "deb http://packages.ros.org/ros2-testing/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros2-latest.list`,
```

Comment:
To reduce duplication, instead of using an if-block here, can we just do something like

```suggestion
  		`echo "deb http://packages.ros.org/ros2${use_ros2_testing ? "-testing" : ""}/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros2-latest.list`,
```